src/client/app.coffee

```diff
@@ -128,7 +44,7 @@ class Arena
             randomPoint = @map.randomPoint()
             randomTeam = _.sample((name for name, r of @teams))
 
-            @focusedUIPlayer = new UIPlayer @, @handler, randomPoint, randomTeam
+            @focusedUIPlayer = new UIPlayer this, @handler, randomPoint, randomTeam
```

Comment:
I was using the style guide here https://github.com/polarmobile/coffeescript-style-guide#naming_conventions and it says:

Prefer @property over this.property.
return @property # Yes
return this.property # No

However, avoid the use of standalone @:
return this # Yes
return @ # No

I can change it back if you like?